### PR TITLE
initial 0.13.0 version

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
anyconfig 0.13.0 for ❄️

- [cf](https://github.com/conda-forge/anyconfig-feedstock)
- [upstream](https://github.com/ssato/python-anyconfig)
- some refactoring compared to conda forge
- added tests
- add more [runtime dependencies](https://github.com/ssato/python-anyconfig/blob/next/requirements.txt)

### CODE REVIEW

**Review by cbouss — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,29 +10,41 @@ source:
...
     - setuptools
-    - python >=3.6
-
+    - Jinja2
+    - PyYAML
+    - jmespath
+    - jsonschema
+    - ruamel.yaml
+    - simplejson
+    - toml
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:32]: These are all optional, see https://github.com/ssato/python-anyconfig/blob/RELEASE_0.13.0/setup.cfg#L71-L81.

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:32]: without those the tests are failing

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:32]: Unit tests usually cover a big part of a package. So I'm not surprised if they fail. But that doesn't mean we should necessarily make the optional dependencies mandatory...

**Pull request merged**